lib/iamport.rb

```
@@ -22,8 +22,8 @@ def config
     def token
       url = "#{IAMPORT_HOST}/users/getToken"
       body = {
-          imp_key: config.api_key,
-          imp_secret: config.api_secret
+        imp_key: config.api_key,
+        imp_secret: config.api_secret,
       }
 
       result = HTTParty.post url, body: body
```

[IMO] If `result["response"]` is `nil`, there will be `undefined method [] for NilClass` error. We should raise a custom error. E.g Iamport::AuthenticationError for that case